src/RokuDeploy.spec.ts

```diff
@@ -3748,85 +3090,25 @@ describe('index', () => {
                 return req;
             });
 
-            const finalPromise = rokuDeploy['getToFile']({}, s`${tempDir}/out/something.txt`).then(() => {
-                getToFileIsResolved = true;
+            const finalPromise = rokuDeploy['downloadFile']({}, s`${tempDir}/out/something.txt`).then(() => {
+                downloadFileIsResolved = true;
             });
 
             await requestCalled.promise;
-            expect(getToFileIsResolved).to.be.false;
+            expect(downloadFileIsResolved).to.be.false;
 
             const callback = await onResponse.promise;
             callback({ statusCode: 200 });
             await util.sleep(10);
 
-            expect(getToFileIsResolved).to.be.false;
+            expect(downloadFileIsResolved).to.be.false;
 
             const writeStream = await writeStreamPromise;
             writeStream.write('test');
             writeStream.close();
 
             await finalPromise;
-            expect(getToFileIsResolved).to.be.true;
-        });
-    });
-
-    describe('deployAndSignPackage', () => {
```

Comment:
Restore these tests. Might need to test the ExecCommand to test these flows

Reply:
Done